lnwallet/btcwallet/btcwallet.go

```diff
@@ -1359,6 +1359,10 @@ func minedTransactionsToDetails(
 		}
 		txDetail.Value = balanceDelta
 
+		if tx.Broadcast != nil {
```

Comment:
@hectorchu in which case is the incorrect rawtx given?

Reply:
For MWEB txns. Because after we've broadcast it we modify the MsgTx to add 'in the clear' MWEB inputs and outputs to the MsgTx, from the MsgTx.Mweb data. So the serialized tx changes. We save the original broadcast serialization in a separate field.